uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/Helper/Menu/ParserTest.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+use CRM_HRCore_Test_BaseHeadlessTest as BaseHeadlessTest;
+use CRM_HRCore_Menu_Navigation as Navigation;
+use CRM_HRCore_Helper_Menu_Parser as MenuParser;
+
+/**
+ * Class CRM_HRCore_Helper_Menu_ParserTest
+ *
+ * @group headless
+ */
+class CRM_HRCore_Helper_Menu_ParserTest extends BaseHeadlessTest {
+
+  public function testGetNavigationTreeReturnsMenuItemsInExpectedFormat() {
+    $navigation = $this->prophesize(Navigation::class);
```

Comment:
With tests that deal with large data sets like this I sometimes find it helpful to put the data in separate files, for example a PHP file that returns an array, or a JSON file that you can use `json_decode` to get the PHP array from

Reply:
Good idea, just that since the data set is not too large and besides there's the advantage of viewing the whole test at once an understanding what is going on without opening other files.

Reply:
Sure, for now it's fine. I'm just thinking if you have a few different test cases in the future and the test file ends up being hundreds of lines long